tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/s2n_tls.h"
+/* To test static functions */
+#include "tls/s2n_server_new_session_ticket.c"
+
+#define ONE_HOUR_IN_NANOS   3600000000000
+#define TWO_HOURS_IN_NANOS  ONE_HOUR_IN_NANOS * 2
+#define TICKET_AGE_ADD_MARKER sizeof(uint8_t)  + /* message id  */ \
+                              SIZEOF_UINT24    + /* message len */ \
+                              sizeof(uint32_t)   /* ticket lifetime */
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+    
+    /* s2n_tls13_server_nst_send */
+    {
+        /**
+         *= https://tools.ietf.org/rfc/rfc5869#appendix-A.1
+         *# PRK  = 0x077709362c2e32df0ddc3f0dc47bba63
+         *#        90b6c73bb50f9c3122ec844ad7c2b3e5 (32 octets)
+         **/
+        S2N_BLOB_FROM_HEX(ticket_key,
+        "4ecd0eb6ec3b4d87f5d6028f922ca4"
+        "c5851a277fd41311c9e62d2c9492e1c4f3");
```

Comment:
These values don't seem to line up

Reply:
Was so concerned with getting the comment correct I forgot about the actual value 😄